src/containers/Form/index.tsx

```diff
@@ -5,30 +5,25 @@ import polygon2 from '../../assets/Polygon 2.png'
 import polygon3 from '../../assets/Polygon 3.png'
 import { Form } from '../../components/Form'
 import { Paper } from '@mui/material'
+import { CardMedia, useTheme } from '@mui/material'
 
 const FormPage = ({ toggleSnackbar, setSnackbarMessage, setLoading }: IHomeProps) => {
   return (
     <>
-      <img src={polygon1} alt='' style={{ position: 'absolute', top: '3%', left: '-10%' }} />
-      <img
-        src={polygon2}
-        alt=''
-        style={{
-          position: 'absolute',
-          top: '50%',
-          right: '20%',
-          transform: 'translateY(-50%)'
-        }}
+      <CardMedia
```

Comment:
Instead of writing duplicate code, I could move this to a different component and then import it into Login/Register/Form(index)